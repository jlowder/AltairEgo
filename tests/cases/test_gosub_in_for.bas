10 FOR I = 1 TO 3
20 PRINT "I=";I: GOSUB 100
30 NEXT I
40 END
100 PRINT "  INSIDE SUBROUTINE"
110 RETURN
