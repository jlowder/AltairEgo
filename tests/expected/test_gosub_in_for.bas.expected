Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
I= 1
  INSIDE SUBROUTINE
I= 2
  INSIDE SUBROUTINE
I= 3
  INSIDE SUBROUTINE
OK
